### PR TITLE
Templates applications

### PR DESCRIPTION
**Pull request by alejandrodob**

### CODE REVIEW

File: templates/applications/app.html [hunk 1]
```diff
@@ -11,12 +11,12 @@
...
     <!-- Application -->
     <div class="col-md-9" style="min-height:400px">
-        {{ helper.render_app_title(app)}}
+        {{ helper.render_app_title(app, upload_method) }}
         <hr>
         {{ helper.render_overall_progress(app, n_tasks, overall_progress) }}
         {% if app.long_description %}
         <hr>
-            {{ app.long_description|safe}}
+            {{ app.long_description | e | markdown }}
```
Comment (teleyinex) [inline templates/applications/app.html:19]: This is just a comment, not something we've to modify ;-) We should warn them that they can enforce markdown for their projects if they want. We keep it like this and disable it if they don't want it.

File: templates/applications/app.html [hunk 2]
```diff
@@ -28,9 +28,9 @@
     </div>
     {% else %}
     <div class="page-header">
-        <h1>{{ _('Sorry! This app does not exists.') }}</h1>
+        <h1>{{ _('Sorry! This app does not exist.') }}</h1>
```
Comment (teleyinex) [inline templates/applications/app.html:31]: exists

File: templates/applications/export.html [hunk 3]
```diff
@@ -10,18 +10,18 @@
...
-        {{ helper.render_app_title(app, subtitle=_('Export All Tasks and Task Runs'))}}
+        {{ helper.render_app_title(app, upload_method, subtitle=_('Export All Tasks and Task Runs'))}}
         <p>{{ _('You can export and download') }} <strong>{{ _('all the available') }}</strong> {{ _('Tasks and Task Runs (the submitted answers by the users) in CSV or JSON formats.') }}</p>
         <div class="row">
             <div id="csv" class="col-md-5 well">
-                <h2>Export in CSV format</h2>
-                <a href={{url_for('app.export_to', short_name=app.short_name, type='task', format='csv')}} class="btn btn-lg btn-primary">{{ _('Tasks') }}</a>
-                <a href={{url_for('app.export_to', short_name=app.short_name, type='task_run', format='csv')}} class="btn btn-lg btn-primary">{{ _('Task Runs') }}</a>
+                <h2>{{_('Export in CSV format')}}</h2>
+                <a href={{url_for('app.export_to', short_name=app.short_name, type='task', format='csv')}} rel="nofollow" class="btn btn-lg btn-primary" download>{{ _('Tasks') }}</a>
```
Comment (teleyinex) [inline templates/applications/export.html:18]: is the download keyword correct here?

Comment (alejandrodob) [inline templates/applications/export.html:18]: It is on our default theme. I guess @therealmarv worked on it (I may be wrong, though) and is for web crawlers related stuff or something similar

**Comment by teleyinex** [comment]
:cake: from my side ;-) after answering my questions :D

File: templates/applications/app.html (hunk 2 above)
Comment (alejandrodob) [inline templates/applications/app.html:31]: does not already includes the third person, so exist is ok

Comment (teleyinex) [inline templates/applications/app.html:31]: :facepalm:

File: templates/applications/export.html (hunk 3 above)
Comment (teleyinex) [inline templates/applications/export.html:18]: ok, so we keep it.

**Comment by teleyinex** [comment]
Markdown :-)

**Pull request merged**